simplic-user/project.repocli.json

```diff
@@ -0,0 +1,14 @@
+{
+  "commands": [
```

Comment:
This is only required for plugins, not yet for framework stuff.

Reply:
Done